controllers/storagecluster/exporter.go

```diff
@@ -83,6 +84,14 @@ func (r *StorageClusterReconciler) enableMetricsExporter(
 		return err
 	}
 
+	// bind the exporter to hostnetwork-v2 SCC if needed
+	if getRookCephHostNetwork(instance) {
```

Comment:
Please add comment for this one and separate this change to a new commit with more details for this change